google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -264,7 +566,7 @@ public long getTimeMillis() {
 
     @Override
     void populateCondition(Rule.Condition condition) {
-      condition.setCreatedBefore(new DateTime(timeMillis));
+      condition.setCreatedBefore(new DateTime(true, timeMillis, 0));
```

Comment:
Is this a breaking change?

Reply:
No, this is a bug fix. It's actually required that it be dateOnly because the time field can't be parsed. Without this, created before never actually works.

Reply:
Thanks for clarifying! Acking.